client/app/telecom/pack/resiliation/pack-resiliation.less

```diff
@@ -1,28 +1,16 @@
 .telecom-pack-resiliation {
 
-    .service-type {
-        margin-bottom: 16px;
-
-        .service-item {
-            .service-select-all {
-                background-color: @gray-light;
-            }
-
-            .service-select-single {
-                padding: 4px 0;
+    .service-select-all {
+        background-color: @gray-light;
+    }
 
-                strong {
-                    color: @oui-color-primary;
-                }
-            }
+    .service-select-single {
+        strong {
+            color: @oui-color-primary;
```

Comment:
You shoud use `.text-primary`

http://getbootstrap.com/docs/3.3/css/#helper-classes-colors